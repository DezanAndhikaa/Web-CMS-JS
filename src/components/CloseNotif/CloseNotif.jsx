import React from 'react';
import './CloseNotif.scss';

export default class CloseNotif extends React.PureComponent {
  render() {
    return (
      <div className="close-btn-row">
        <div className="close-btn" onClick={this.props.onClose} />
      </div>
    );
  }
}
