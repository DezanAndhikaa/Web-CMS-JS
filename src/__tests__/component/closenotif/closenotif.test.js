import React from 'react';
import Enzyme, { shallow } from "enzyme";
import Adapter from 'enzyme-adapter-react-16';
import CloseNotif from '../../../components/CloseNotif/CloseNotif';

Enzyme.configure({adapter:new Adapter()});
it("render button close notif correctly", () => {
  const wrapper = shallow (
    <CloseNotif />
  )
  expect(wrapper).toMatchSnapshot();
})