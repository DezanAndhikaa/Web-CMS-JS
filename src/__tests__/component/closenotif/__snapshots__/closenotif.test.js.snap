// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render button close notif correctly 1`] = `ShallowWrapper {}`;
